polaris/evaluate/_metric.py

```diff
@@ -30,6 +31,30 @@ def spearman(y_true: np.ndarray, y_pred: np.ndarray):
     return stats.spearmanr(y_true, y_pred).statistic
 
 
+def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray):
+    """
+    Calculate the Absolute Average Fold Error (AAFE) metric.
+    It measures the fold change between predicted values and observed values.
+    The implementation is based on https://pubs.acs.org/doi/10.1021/acs.chemrestox.3c00305.
```

Comment:
Looks nice in the rendered docs! 

```suggestion
    The implementation is based on [this paper](https://pubs.acs.org/doi/10.1021/acs.chemrestox.3c00305).
```